boa/vyper/ast_utils.py

```diff
@@ -6,9 +6,13 @@
 from vyper.codegen.core import getpos
 
 
-def get_line(source_code: str, lineno: int) -> str:
+def get_block(source_code: str, lineno: int, end_lineno: int) -> str:
```

Comment:
This is introducing a breaking change for the line profiler: https://github.com/benber86/titanoboa/blob/c537ea1d9f1d61bf56288ab24ae1d99c77a3c3a8/boa/profiling.py#L121

So you'll need to address this. I think the block idea is good because line profiler can look quite ugly if the lines are multi-line.

Reply:
I didn't remove `get_line`, just moved it two lines down and refactored to use `get_block`:

```
def get_line(source_code: str, lineno: int) -> str:
    return get_block(source_code, lineno, lineno)
```